vocab/credentials/vocab.csv

```diff
@@ -5,6 +5,8 @@ odrl,prefix,,http://www.w3.org/ns/odrl/2/,,,,,
 xsd,prefix,,http://www.w3.org/2001/XMLSchema#,,,,,
 ,rdfs:seeAlso,,https://www.w3.org/TR/vc-data-model/,,,,,
 JsonSchemaValidator2018,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+JsonSchema2023,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+CredentialSchema2023,rdfs:Class,,,,,,,A type of validator that can be used to syntactically validate JSON documents packaged as Verifiable Credentials using the JSON Schema language.
```

Comment:
```suggestion
```

This is modifying the v1 CSV vocabulary, which we shouldn't be doing for a v2 feature.